pygsheets/worksheet.py

```diff
@@ -1267,6 +1267,45 @@ def copy_to(self, spreadsheet_id):
         new_spreadsheet = self.client.open_by_key(spreadsheet_id)
         return new_spreadsheet[response['index']]
 
+
+    def sort_range(self,start,end,basecolumnindex,sortorder):
+        """Sorts the data in rows based on the given column index.
+
+        :param start:               Address of the starting cell of the grid (e.g.-> "A1") 
```

Comment:
either remove the example or add an eg that it can be tuple also, current eg makes it look like it can only be label.